examples/tts/conf/audio_codec/audio_codec_44100.yaml

```diff
@@ -0,0 +1,194 @@
+# This config contains the default values for training 44.1kHz audio codec model which encodes mel spectrogram
+# instead of raw audio.
```

Comment:
Is this mel spectrogram codec or raw audio codec?

Reply:
Changed description to raw audio codec.